### ISSUE TITLE
Packet custom ipxe chain load loop

### ISSUE THREAD

**Issue by mikemrm** [netbootxyz/netboot.xyz#495]
**Describe the bug**
Howdy, it seems the ipxe image for packet wasn't updated when the switching to version 2.x as it seems to load netboot.xyz-packet.efi (or .kpxe) show it's version 1.04 and then attempt to download version 2.x again and continues to do that indefinitely.

This is happening on both uefi and legacy booting systems.

**To Reproduce**
1. Login to app.packet.com
2. Deploy a server with custom ipxe set to https://boot.netboot.xyz
3. Watch SOS

**Expected behavior**
Load netboot.xyz image and display menus

**SOS Output**
```
Packet.net Baremetal - iPXE boot
http://139.178.81.6/phone-home... ok
https://boot.netboot.xyz... ok
https://boot.netboot.xyz/menu.ipxe... ok
boot.cfg... ok
Attempting to retrieve latest upstream version number...
https://boot.netboot.xyz/version.ipxe... ok

Updated version of netboot.xyz is available:

Running version.....1.0.255+ (18dc7)
Updated version.....2.x

Please download the latest version from netboot.xyz.

Attempting to chain to latest version...
http://boot.netboot.xyz/ipxe/netboot.xyz-packet.efi... ok
iPXE initialising devices...ok

iPXE 1.20.1+ (g18dc) -- Open Source Network Boot Firmware -- http://ipxe.org
Features: DNS HTTP HTTPS iSCSI TFTP SRP VLAN AoE EFI Menu
netboot.xyz v1.04 for packet.com
                                      
Configuring (net0 98:03:9b:2f:3c:ea).................. ok
https://boot.netboot.xyz/menu.ipxe... ok
boot.cfg... ok
Attempting to retrieve latest upstream version number...
https://boot.netboot.xyz/version.ipxe... ok

Updated version of netboot.xyz is available:

Running version.....1.04
Updated version.....2.x

Please download the latest version from netboot.xyz.

Attempting to chain to latest version...
http://boot.netboot.xyz/ipxe/netboot.xyz-packet.efi... ok
iPXE initialising devices...ok

iPXE 1.20.1+ (g18dc) -- Open Source Network Boot Firmware -- http://ipxe.org
Features: DNS HTTP HTTPS iSCSI TFTP SRP VLAN AoE EFI Menu
netboot.xyz v1.04 for packet.com
                                      
Configuring (net0 98:03:9b:2f:3c:ea).................. ok
https://boot.netboot.xyz/menu.ipxe... ok
boot.cfg... ok
Attempting to retrieve latest upstream version number...
https://boot.netboot.xyz/version.ipxe... ok

Updated version of netboot.xyz is available:

Running version.....1.04
Updated version.....2.x

Please download the latest version from netboot.xyz.

Attempting to chain to latest version...
http://boot.netboot.xyz/ipxe/netboot.xyz-packet.efi... ok
iPXE initialising devices...ok
...
```

**Comment by antonym** [comment]
Thanks for the ticket, I ran across this a few days ago and submitted the fix:

https://github.com/netbootxyz/netboot.xyz/commit/5d09e0eedec335a47532e183f0c72f88c73e1c97#diff-f34195b187d245f864478fe0a9b0c70c

It should be released this week in 2.0.4.

**Comment by mikemrm** [comment]
Ah, should've done a blame on those. I looked at those files and everything seemed right. Thanks!

**Comment by antonym** [comment]
Released 2.0.4 and tested on Packet and it looks like it's working now.  Closing out and thanks!
